app/src/main/java/com/google/droidjump/GameView.java

```diff
@@ -132,11 +143,26 @@ private void checkTimePoint() {
     private void updateGameState() {
         checkTimePoint();
         updateDroidCoordinates();
+        updateObstaclesCoordinates();
+        updatePlatformCoordinates();
+        // Level Finishing
         if (timePoint == levelTimePoints) {
             winGame();
         }
     }
 
+    private void updateObstaclesCoordinates() {
+        Iterator<Obstacle> it = obstacleList.iterator();
+        while (it.hasNext()) {
+            Obstacle obstacle = it.next();
+            obstacle.setX(obstacle.getX() - levelSpeed);
+
+            // Removal of passed obstacles
+            if (obstacle.getX() + obstacle.getWidth() < 0)
+                it.remove();
```

Comment:
nit: for if/while/else/for always have brackets {}
if (obstacle.getX() + obstacle.getWidth() < 0) {
                it.remove();
}